readme.md

```diff
@@ -1,8 +1,12 @@
 Nodehun
 =======
+[![Build Status](https://travis-ci.org/Wulf/nodehun.svg?branch=master)](https://travis-ci.org/Wulf/nodehun) Unix
+
+[![Build status](https://ci.appveyor.com/api/projects/status/9ky5lws4d191qrui/branch/master?svg=true)](https://ci.appveyor.com/project/Wulf/nodehun/branch/master) Windows
+
```

Comment:
Drop the `unix` and `windows`, people can infer that from `travis` and `appveyor`